docs/getting-started/cluster-setup.md

```diff
@@ -158,3 +159,12 @@ To view the stdout / stderr from a pod, run:
 ```bash
 $ kubectl logs <POD-NAME>
 ```
+
+## Update your OAuth Application
+You will need to navigate back to your original OAuth Application that you used for your OAuth Client ID and Secret to update the URLs.
+
+1. Navigate to the [OAuth applications](https://github.com/settings/developers) page.
+
+2. Click on the application you created to get your OAuth Client ID and Secret.
+
+3. Fill out the `Homepage URL` and `Authorization callback URL` with your `LoadBalancer Ingress`.
```

Comment:
Is it just the `http://load-balancer.aws.com` , or is there a specific route to hit?